src/session.js

```diff
@@ -12,7 +12,6 @@ export function getClientMetadataID(): ?string {
 function getSDKStorage(): Storage {
   return getStorage({
     name: getNamespace(),
-    stickySessionId: getClientMetadataID() || "",
```

Comment:
This was showing as a Flow error as getStorage doesn't have a param for `stickySessionId`. I do have a worry that I'm missing something with how our build system might _actually_ make it so it does lol @wsbrunson am I missing anything in that regard?

Reply:
do you have the latest version of `belter`?

Reply:
Oh weird, I do but I think when I initially did this Belter was outa date. But I've updated this now, thanks!